kumascript/macros/GroupData.json

```diff
@@ -2097,35 +2097,15 @@
       "methods": ["WebGLRenderingContext.makeXRCompatible()"],
       "properties": ["Navigator.xr"],
       "events": [
-        "XR: devicechange",
+        "XRSystem: devicechange",
```

Comment:
I think that events children of one of the interfaces listed under "interfaces" automatically included in the sidebar (if they have the tag `Event` set).

We should list only special events, like we list special properties or methods.

I have not double-checked but I have fixed numerous sidebar in July without adding the events here.

In other words, I think that we should empty the `events` section here.

Reply:
I can't confirm. When I empty the events list, the events disappear from the sidebar on http://localhost:3000/en-US/docs/Web/API/WebXR_Device_API

Reply:
Let me experiment.

Reply:
Were you able to test this?

Reply:
Sorry, I missed the comment. What you do is correct!